docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -231,6 +241,84 @@ Each test run has its own test context.
 
 > `t.ctx` is initialized with an empty object without a prototype. You can iterate its keys without the `hasOwnProperty` check.
 
+### Fixture Hooks
+
+Fixture hooks are executed before the first test in a fixture is started and after the last test is finished.
+
+Unlike [test hooks](#test-hooks), fixture hooks are executed when tests are not running.
```

Comment:
> when tests are not running

I don't like this part, sounds like they run then all tests are stalled. It's better be described as hooks that run in between tests and they don't have access to tested page.